assets/src/application/ui/display/confirm/useConfirmationDialog.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useCallback, useMemo } from 'react';
-import { useDisclosure } from '@chakra-ui/core';
+import { useDisclosure } from "@chakra-ui/hooks"
```

Comment:
When did we switch to double quotes? 🤔

Reply:
my prettier is not working as expected, will reinstall VS Code and it's plugins

Reply:
Oh OK